spec/requests/cats_spec.rb

```diff
@@ -3,30 +3,52 @@
 RSpec.describe "Cats", type: :request do
   describe "GET /index" do
     it "gets a list of cats" do
-    Cat.create name: 'Methusal', age: 3, enjoys: 'Eating Pickles'
-    get '/cats'
-    cat = JSON.parse(response.body)
-    expect(response).to have_http_status(200)
-    expect(cat.length).to eq 1
-   end
-  end
+     Cat.create( 
+       name: 'Giraffe', 
+       age: 3, 
+       enjoys: 'Eating tuna',
+       image: 'https://www.indeed.com/'
+      )
+       get '/cats'
+       cat = JSON.parse(response.body)
+       expect(response).to have_http_status(200)
+       expect(cat.length).to eq 1
+      end
+    end
   describe "POST /create" do
     it "creates a cat" do
-
       cat_params = { 
         cat: {
           name: 'Giraffe',
-          age: 5, 
-          enjoys: 'Looking at trees'
+          age: 3, 
+          enjoys: 'Eating tuna',
+          image: 'https://townsquare.media/site/396/files/2021/06/attachment-Cat-Walking-On-Beach.jpg'
         }
       }
       post '/cats', params: cat_params
       expect(response).to have_http_status(200)
-
       cat = Cat.first
-      expect(cat.name).to eq 'Giraffe'
-      expect(cat.age).to eq 5
-      expect(cat.enjoys).to eq 'Looking at trees'
+      expect(cat.name).to eq('Giraffe')
+      expect(cat.age).to eq(3)
+      expect(cat.enjoys).to eq('Eating tuna')
+      expect(cat.image).to eq('https://townsquare.media/site/396/files/2021/06/attachment-Cat-Walking-On-Beach.jpg')
+    end
+  end
+
+  describe "can request validation" do
```

Comment:
Much better naming convention!  I don't see request validations for age, enjoys, and image.